include/storage/io.hpp

```diff
@@ -104,6 +104,13 @@ class FileReader
         }
     }
 
+    void ReadInto(std::string &target)
+    {
+        auto len = ReadElementCount32();
```

Comment:
I'm using this particular method for osm strings, but it's not unlikely that this method could be reused in the future for strings other than osm strings. Would a 16 bit size still work?